src/main/java/com/training/utilities/GetJsonValue.java

```diff
@@ -23,15 +22,8 @@ public String getValue(String key)  {
         } catch (IOException e) {
             e.printStackTrace();
         }
-
         // convert JSON file to map
         map = gson.fromJson(reader, Map.class);
-
-        // print map entries
-        for (Map.Entry<?, ?> entry : map.entrySet()) {
-            if(key.equalsIgnoreCase((String) entry.getKey()))
-            return (String) entry.getValue();
-        }
-        return null;
+        return map;
```

Comment:
instead of creating a variable map, return directly - gson.fromJson(reader, Map.class);